### PR TITLE
chore: set max SDK version to <3.0.0

### PR DESCRIPTION
**Pull request by chalin**
cc @kwalrath @kevmoo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.9.2+1
```
Comment (nex3) [inline CHANGELOG.md]: 1.9.3

We only use build identifiers for pre-1.0 packages.

File: pubspec.yaml [hunk 2]
```diff
@@ -7,14 +7,17 @@ name: stack_trace
 #
 # When the major version is upgraded, you *must* update that version constraint
 # in pub to stay in sync with this.
-version: 1.9.2
-author: "Dart Team <misc@dartlang.org>"
-homepage: https://github.com/dart-lang/stack_trace
+version: 1.9.2+1
```
Comment (nex3) [inline pubspec.yaml]: Also 1.9.3

**Review by nex3 — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 2 above)
Comment (chalin) [reply pubspec.yaml]: Both done

**Comment by chalin** [comment]
@nex3 @kevmoo - could you restart the firefox job? It timed out.

**Comment by kevmoo** [comment]
Done!

**Review by nex3 — APPROVED** [review]

**Pull request merged**